### PR TITLE
completed array-manipulation-1

### PR DESCRIPTION
**Pull request by Yi-Jacob**
Screen Shot 2022-02-04 at 8 03 26 PM

### CODE REVIEW

File: array-manipulation-1/src/initial.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+function with parameter of array
+take in an array as an argument
+get the first element of array
+remove the first element
+return the new array without the first element
+*/
+function initial(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== array.length - 1) {
```
Comment (codingcodymiller) [inline array-manipulation-1/src/initial.js]: If you stop the loop one index sooner, you won't need this conditional 🤓

File: array-manipulation-1/src/tail.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+function with parameter of array
+take in an array as an argument
+get the first element of array
+remove the first element
+return the new array without the first element
+*/
+function tail(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== 0) {
```
Comment (codingcodymiller) [inline array-manipulation-1/src/tail.js]: Same here, if you start the loop at 1, you won't need a conditional

File: array-manipulation-1/src/reverse.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+run the function for the length of the array
+remove the items from the old array from the LAST element
+add the items from the old array to the new array
+return the array
+*/
+
+function reverse(array) {
+  var newArray = [];
+  while (array.length) {
+    newArray.push(array.pop());
```
Comment (codingcodymiller) [inline array-manipulation-1/src/reverse.js]: By using `pop`, you are changing the original array, which is a serious problem. Please complete this without making changes to the original array

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-02-07 at 1 23 43 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**